### ISSUE TITLE
Unable to donate to this project :sadpanda:

### ISSUE THREAD

**Issue by waterlink** [Codcore/Amethyst#35]
pledgie returns this error, when trying to donate using PayPal:

```
The email account Receiver not in receive money enabled country is based in a country that isn't enabled to receive payments
```

**Comment by Codcore** [comment]
@waterlink, thank you for a feedback, it seems that there is some issues with receiving payments to my country :( . Anyway, you can use webmoney, my WMZ Z735788749072

**Comment by waterlink** [comment]
I am in Berlin, can I use webmoney?

---

That would be nice to have a counter on the badge. Because it will stay zero even if someone donates through webmoney. This might discourage people from donating..

Could be there is different service that allow payments to go through in your country?

**Comment by Codcore** [comment]
@waterlink yes, you can, I checked, and Google says that it is no restriction for Germany

**Comment by waterlink** [comment]
Heh, that is ridiculous. If I want to convert 20 Paypal bucks to webmoney, the fee is 15 bucks.. And it is only a conversion that should be done only if you are owner of both accounts, which effectively means, that there will be more commission for actual sending. That definitely doesn't make sense for donations, because they are usually small. This is a service I am trying to use: https://goo.gl/XH5J7l

**Comment by Codcore** [comment]
@waterlink here is exchanger with smallest fee I could find. But you should understand Russian language. Maybe you can propose other payment system?

**Comment by Codcore** [comment]
@waterlink Actually, my email at PayPal was not confirmed, maybe this caused error. I confirmed email, can you try once more?

**Comment by waterlink** [comment]
Didn't help, the same error about country having payment not enabled. I think that is how paypal works in some countries: you can pay, but you cannot receive money.

I don't know donating systems very well (or actually almost at all). Can you send the link to the exchanger by email (you can find it on my public https://github.com/waterlink profile).

**Comment by oprypin** [comment]
PayPal simply won't let you receive money in Ukraine. Really is a tough situation with these things here.
